py/server/tests/test_column.py

```diff
@@ -136,25 +136,29 @@ def test_datetime_col(self):
         inst = dhtime.to_j_instant(round(time.time()))
         dt = datetime.datetime.now()
         _ = datetime_col(name="Datetime", data=[inst, dt, None])
-        self.assertEqual(_.data_type, dtypes.Instant)
+        self.assertEqual(_.name, "Datetime")
+        self.assertEqual(_.j_column.type(), dtypes.Instant.qst_type)
```

Comment:
What happened to `.data_type` in this case? Looks like a breaking change?

Reply:
You mentioned earlier that we didn't need `InputColumn` to be an instance of `Column`; so, it is "losing" the attributes `data_type`, `component_type`, `column_type`, `j_column_header`. I maintain that they should never have been there in the first place (the inheritance doesn't make sense IMO).